src/services/usageTracking.ts

```
@@ -0,0 +1,497 @@
+  messageLength: number,
+  recipientCountryCode: string = '+1'
+): Promise<{
+  estimatedSegments: number;
+  estimatedCost: number;
+  currency: string;
+}> {
+  // GSM-7 encoding: 160 chars per segment (153 for multi-segment)
+  // Unicode: 70 chars per segment (67 for multi-segment)
+  
+  // Simple estimation - actual implementation would check encoding
+  const isUnicode = /[^\x00-\x7F]/.test(''); // Simplified check
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)